datamade/erikson-edi
# Use pole of inaccessibility for polygon centroids

## Overview

This PR updates the algorithm we use to calculate centroids for polygons to use a [Mapbox library](https://github.com/mapbox/polylabel) for calculating the pole of inaccessibility.

## Testing instructions

I tested this for Kankakee and GESTL, which are the only two partner communities I have imported so far.

- Spin up the dev server
- Visit the map view for each partner community 
- Select a data overview and confirm that the centroids look good
